falcon-regression/merlin/src/test/java/org/apache/falcon/regression/hcat/HCatFeedOperationsTest.java

```diff
@@ -119,7 +119,7 @@ public void tearDownClass() throws IOException {
      *
      * @throws Exception
      */
-    @Test(groups = {"singleCluster"})
+    @Test(groups = {"multiCluster"})
```

Comment:
@raghavgautam  submitFeedWhenTableDoesNotExist uses second cluster information to submit the feed. Since we are separating the execution of single vs multi cluster test cases, this will come under multiCluster tag.